db/migrate/20220802162516_update_prize_lead_foreign_key.rb

```diff
@@ -0,0 +1,5 @@
+class UpdatePrizeLeadForeignKey < ActiveRecord::Migration[7.0]
+  def change
+    rename_column :leads, :prizes_id, :prize
```

Comment:
The renamed column should be `:prize_id`. That way Rails knows how to do its magic with the associations

This is probably why Rails was complaining when you tried to add the `belongs_to` in the `Lead` model

Reply:
![image](https://user-images.githubusercontent.com/62039765/182475186-94b2d178-1427-4095-95d8-c3a929b4499b.png)
It still gives the same error even after doing this fix.

Reply:
You can add an `optional: true` to the `belongs_to` statement

![image](https://user-images.githubusercontent.com/2940022/182476594-0a3eaea4-b737-4909-9a55-e4e7427ce582.png)

Reply:
That makes sense, thanks!!